docs/source/usage/automatic-sql-registration.rst

```diff
@@ -14,7 +14,7 @@ Pre-requisites
 **************
 
 In order to use Mosaic, you must have access to a Databricks cluster running
-Databricks Runtime 10.0 or later. If you have cluster creation permissions in your Databricks
+Databricks Runtime 11.2 or later. If you have cluster creation permissions in your Databricks
```

Comment:
@edurdevic will it still work before 11.2 with the fallback?
If yes, I would say 10.0 is prerequisite, 11.2 is recommended.
Maybe we need to reword this sentence.

Reply:
Yes, you are right! 
I wrote this comment before we decided to include the H3 JAR in Mosaic as well.